cmd/kube-apiserver/app/server.go

```diff
@@ -460,32 +461,30 @@ func (s *APIServer) Run(_ []string) error {
 		glog.Fatalf("Invalid server address: %v", err)
 	}
 
-	legacyV1Group, err := latest.Group("")
-	if err != nil {
-		return err
+	if !latest.IsEnabled() {
```

Comment:
Yes, this code should be looping over all available versions. You should *not* have to modify this file to start serving a new group.